unitTests/ui/ValueElementRendererTest.ts

```diff
@@ -28,6 +28,11 @@ export function ValueElementRendererTest() {
       expect(valueRenderer.build().listItem.getAttribute('data-value')).toBe('foo');
     });
 
+    it('should not make the label a div element', () => {
```

Comment:
The test name (or a comment inside the test) should describe/explain why it should not be a label.

Reply:
Woops, first of all, it looks like I inverted the meaning of this test's name.

And that's a good point, I'll change this test's name to `should give the label the "section" role to avoid overriding child labels`